### PR TITLE
feat: add support for Kubernetes v1.18.10

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#changelog-since-v1189

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
Had to rebase after #3937 merged. Also the WS2019 VHD - docker job had a failure, so hopefully this time it's happier. 🤞

File: vhd/packer/install-dependencies.sh [hunk 1]
```diff
@@ -366,7 +366,6 @@ K8S_VERSIONS="
 1.19.3
 1.19.2
 1.18.9
```
Comment (jackfrancis) [inline vhd/packer/install-dependencies.sh:368]: Did we forget to add 1.18.10 here?

Comment (mboersma) [reply vhd/packer/install-dependencies.sh:368]: D'oh, rebase failure. I'll fix now.

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**